spec/system/distribution_system_spec.rb

```diff
@@ -264,7 +264,7 @@
 
         expect(page).to have_css "td"
         item_row = find("td", text: diaper_type).find(:xpath, '..')
-        expect(item_row).to have_content("#{diaper_type} 4")
+        expect(item_row).to have_content(diaper_type, ' 4')
```

Comment:
Short Answer:

You're right. I thought I would be testing for both strings, and didn't realize that '4' could be part of the diaper type. What I would really like is to get the string ("i.e. `4T Diapers $1.00 $4.00 4`) and test it myself, either with regex or other Ruby methods. Is there a way I can do that?


Long Answer:

My apologies. When I was working on this, I searched for the docs I found this at https://www.rubydoc.info/github/jnicklas/capybara/Capybara/RSpecMatchers:

```
 #have_text(*args) ⇒ Object (also: #have_content)
RSpec matcher for text content See Node::Matchers#assert_text.
```

I incorrectly assumed that `*args` was merely an array of strings to search for. Since the error was :

`expected to find text "4T Diapers 4" in "4T Diapers $1.00 $4.00 4"
`
...I assumed I could just skip the cost and extension (in case the cost was ever changed, we did not want a false positive) and test for both "4T Diapers" and the final "4"; but when I tested it, the diaper type did not have a 4 in it, so I thought that would work.

I should have clicked to see the detailed documentation, which would have shown me that my usage was wrong.

Reply:
No apology necessary, we have code review for a reason! Thanks for the detailed explanation by the way :)